src/SingleView/SingleView.js

```diff
@@ -2,14 +2,14 @@ import React from "react"
 import DetailContainer from "../DetailContainer/DetailContainer"
 import "./SingleView.css"
 
-const SingleView = (props) => {
+const SingleView = ({ movie, showAll }) => {
   
   return (
     <div className='single-view' style={{
```

Comment:
Consider pulling out this inline styling and putting it in the app.css file. It's not that its "wrong", but there are some good arguments for why some devs think its cleaner and more maintainable to avoid inline styling like this.  Since this backgroundImage isn't dynamic, you should be able to pull it out of the inline styling.